src/features/Chat/components/Menu/index.tsx

```diff
@@ -31,19 +31,21 @@ const Menu = () => {
   const { isTablet } = useGetLayoutMode();
   const { showFolders, activeFolder } = useAppSelector(state => state.chats);
   const chats = useAppSelector(selectChats);
+  const chatsExist = chats.length > 0;
 
   const menuContent = (
     <>
-      <Header showSearch={chats.length > 0} />
+      <Header showSearch={chatsExist} />
       {(showFolders || ['archived', 'banned'].includes(activeFolder)) && (
         <FolderLink targetFolder="ok" />
       )}
-      {showFolders ? (
+      {showFolders && (
         <>
           <FolderLink targetFolder="archived" />
           <FolderLink targetFolder="banned" />
         </>
-      ) : chats.length ? (
+      )}
+      {!showFolders && chatsExist && (
```

Comment:
Changes to remove ternaries are great, like it.
THis is just a comment, not need to act on it on this PR.

When I introduce boolean variables that have multiple conditions for rendering, I like to introduce it before the return and name it properly. Example:

```react
const isTheComponentVisible = resources.length > 0 && !isLoading

return {
isTheComponentVisible && <TheComonent />
}
```

It makes evaluating the stuff easier for the brainz imo. :p

Reply:
I refactored this and think it's now clearer! Thanks!